core/org.osate.ui/src/org/osate/ui/navigator/VirtualProjectDependencies.java

```diff
@@ -0,0 +1,21 @@
+package org.osate.ui.navigator;
+
+import org.eclipse.core.resources.IProject;
+
+public final class VirtualProjectDependencies {
+	private final IProject project;
+	private final IProject[] children;
+
+	public VirtualProjectDependencies(final IProject project, final IProject[] children) {
+		this.project = project;
+		this.children = children;
+	}
+
+	public IProject getProject() {
+		return project;
+	}
+
+	public IProject[] getChildren() {
```

Comment:
Children should not be IProjects but a wrapper class. Currently the depended on projects can be expanded inside the project dependencies node even though they exist as top-level projects in the workspace. I think it would be better to show the project names only.

Reply:
fixed.  Hardest part was finding the correct icon file to use.